src/verifying.rs

```diff
@@ -471,6 +480,12 @@ where
     }
 }
 
+impl From<&VerifyingKey> for MontgomeryPoint {
```

Comment:
I'm a little concerned that this is more confusing than it is helpful. It is already very niche for someone to want to use a sig pubkey as a KEM pubkey. I think the `to_montgomery()` method is enough.

Reply:
I can remove these. I do agree the inherent methods are more explicit.

Reply:
Removed in 239eecb